### ISSUE TITLE
Create DEB package to be included in Debian/Ubuntu

### ISSUE THREAD

**Issue by francescmm** [francescmm/GitQlient#73]
It would be nice if GitQlient was included in the official Ubuntu and Debian repos to install using apt-get.

**Comment by hegjon** [comment]
Depends on #156

**Comment by francescmm** [comment]
Unblocked! :D

**Comment by Morph21** [comment]
How it's  going with that issue? I would like to try it out on linux. Its there any other way to run it on ubuntu?

**Comment by francescmm** [comment]
Hi @Morph21 I didn't get any help so far with the package so I could do anything yet. However you can download the [AppImage from the release](https://github.com/francescmm/GitQlient/releases/download/v1.2.0/GitQlient-1.2.0-x86_64.AppImage) and try it! :smiley:

**Comment by francescmm** [comment]
@band-a-prend In another issue you mention that Debian packages have `recommended` dependencies. That made me think that you could have some experience packaging for Debian. Do you? :smile:

**Comment by band-a-prend** [comment]
No, I didn't package for deb-based distributives, just used Debian + Mint few years before returning to Gentoo.

I remember that for some packages there were [`Recommends` and `Suggests`](https://www.debian.org/doc/debian-policy/ch-relationships.html#binary-dependencies-depends-recommends-suggests-enhances-pre-depends) dependencies. The installation of them is allowed by default and could be managed by apt package manager common configuration or by apt command flags at the moment of pkg installation.

**Comment by francescmm** [comment]
Hi @waldyrious ! No, you're not missing anything. I'm having some problems generating the .deb package (#218). I couldn't find time so far to create one that runs and I'm wondering if it I should just remove that from now :disappointed: 

I hope I can find some time this weekend, though :)

**Comment by francescmm** [comment]
@waldyrious If you want to try the DEB package to see if it works, I've uploaded a [generated package here](https://www.dropbox.com/s/m2xsezlczhn4krh/gitqlient_1.4.0_amd64.deb).

**Comment by waldyrious** [comment]
Yep, it works! Though I'm not sure if something's missing from it, as Ubuntu's software center reports the license as "proprietary" :thinking:

![Screenshot from 2021-08-23 00-12-58](https://user-images.githubusercontent.com/478237/130378697-cc8a6c14-e54b-4697-87a0-703b9c88a172.png)

**Comment by francescmm** [comment]
Yep, it's a matter of missing License file. I'll fix that and and wait for confirmation from Linux Mint.

**Comment by francescmm** [comment]
> Yep, it works! Though I'm not sure if something's missing from it, as Ubuntu's software center reports the license as "proprietary" 
> 
> ![Screenshot from 2021-08-23 00-12-58](https://user-images.githubusercontent.com/478237/130378697-cc8a6c14-e54b-4697-87a0-703b9c88a172.png)

It seems there are a lot of packages having this issue, I'll ship it for now, and keep one eye on it.
